Source/WebCore/html/HTMLFormControlElement.h

```diff
@@ -68,6 +68,9 @@ class HTMLFormControlElement : public LabelableElement, public FormListedElement
     bool wasChangedSinceLastFormControlChangeEvent() const { return m_wasChangedSinceLastFormControlChangeEvent; }
     void setChangedSinceLastFormControlChangeEvent(bool);
 
+    bool wasInteractedSinceLastFormSubmitEvent() const { return m_wasInteractedSinceLastFormSubmitEvent; }
```

Comment:
Common English usage is "interacted with". So `wasInteractedWithSinceLastFormSubmitEvent` perhaps.